src/v0/sources/shopify/util.js

```diff
@@ -151,38 +136,34 @@ const setAnonymousIdorUserIdFromDb = async (message, metricMetadata) => {
       cartToken = properties?.cart_token;
       break;
     /*
-     * we dont have cart_token for carts_create and update events but have id and token field
+     * we dont have cart_token for carts_update events but have id and token field
      * which later on for orders become cart_token so we are fethcing cartToken from id
      */
-    case SHOPIFY_TRACK_MAP.carts_create:
     case SHOPIFY_TRACK_MAP.carts_update:
       cartToken = properties?.id || properties?.token;
       break;
-    // https://help.shopify.com/en/manual/orders/edit-orders -> order can be edited through shopify-admin only
-    // https://help.shopify.com/en/manual/orders/fulfillment/setting-up-fulfillment -> fullfillments wont include cartToken neither in manual or automatiic
+    // order edit and fullfillments events dont have cart_token or id hence shopify-admin 
     default:
   }
   if (!isDefinedAndNotNull(cartToken)) {
-    stats.increment('shopify_no_cartToken', {
+    stats.increment('No_cartToken_in_payload', {
```

Comment:
keep all small no_cart_token_in_payload

what are we planning to do with this metric?

Reply:
we want to know in how many cases will we wont get any `cartToken` in payload